server/routes/money.js

```diff
@@ -155,6 +239,72 @@ const createMoneyRouter = ({
     }
   });
 
+  router.get('/transactions/private', async (req, res, next) => {
+    try {
+      const templateData = {
+        title: 'Your transactions',
+        config: {
+          content: false,
+          header: false,
+          postscript: true,
+          detailsType: 'small',
+          returnUrl: req.originalUrl,
+        },
+      };
+
+      const { user } = req;
+
+      if (user) {
```

Comment:
Not a specific query for this PR but does this only happen when a user is logged out? 

I'm obviously missing a lot of context but it seems like some pages are decorated with additional info when a user is logged in but are still accessible when they aren't. 

Does it make sense that a user can access this page when not logged in? 
Also are links to this page available when a user is not logged in?

Reply:
Probably not for this page, but historically for personal information;
- Signing in has always been optional, prompts to sign in have been on the personal information components themselves
- The user is able to sign out on these pages and we wanted to preserve the their history so avoided redirects
- We've displayed personal information alongside related content, so it made sense to still display the page

But 100% agree this boilerplate is some-what redundant here. I think - and probably not for this PR, it would be worth exploring a more elegant approach with middleware down the line